src/core/public/plugins/plugin_loader.test.ts

```diff
@@ -1,125 +0,0 @@
-
-  // Mock document.body.appendChild to avoid errors about appending objects that aren't `Node`'s
-  // and so we can verify that the script tags were added to the page.
-  appendChildSpy = jest.spyOn(document.body, 'appendChild').mockReturnValue({} as any);
-
-  // Mock global fields needed for loading modules.
-  coreWindow.__kbnBundles__ = {};
-});
-
-afterEach(() => {
-  appendChildSpy.mockRestore();
-  createElementSpy.mockRestore();
-  delete coreWindow.__kbnBundles__;
-});
-
-const addBasePath = (path: string) => path;
-
-test('`loadPluginBundles` creates a script tag and loads initializer', async () => {
-  const loadPromise = loadPluginBundle(addBasePath, 'plugin-a');
-
-  // Verify it sets up the script tag correctly and adds it to document.body
-  expect(createdScriptTags).toHaveLength(1);
-  const fakeScriptTag = createdScriptTags[0];
-  expect(fakeScriptTag.setAttribute).toHaveBeenCalledWith(
-    'src',
-    '/bundles/plugin/plugin-a/plugin-a.plugin.js'
-  );
-  expect(fakeScriptTag.setAttribute).toHaveBeenCalledWith('id', 'kbn-plugin-plugin-a');
-  expect(fakeScriptTag.onload).toBeInstanceOf(Function);
-  expect(fakeScriptTag.onerror).toBeInstanceOf(Function);
-  expect(appendChildSpy).toHaveBeenCalledWith(fakeScriptTag);
-
-  // Setup a fake initializer as if a plugin bundle had actually been loaded.
-  const fakeInitializer = jest.fn();
-  coreWindow.__kbnBundles__['plugin/plugin-a'] = { plugin: fakeInitializer };
-  // Call the onload callback
-  fakeScriptTag.onload();
-  await expect(loadPromise).resolves.toEqual(fakeInitializer);
-});
-
-test('`loadPluginBundles` includes the basePath', async () => {
-  loadPluginBundle((path: string) => `/mybasepath${path}`, 'plugin-a');
-
-  // Verify it sets up the script tag correctly and adds it to document.body
-  expect(createdScriptTags).toHaveLength(1);
-  const fakeScriptTag = createdScriptTags[0];
-  expect(fakeScriptTag.setAttribute).toHaveBeenCalledWith(
-    'src',
-    '/mybasepath/bundles/plugin/plugin-a/plugin-a.plugin.js'
-  );
-});
-
-test('`loadPluginBundles` rejects if script.onerror is called', async () => {
-  const loadPromise = loadPluginBundle(addBasePath, 'plugin-a');
-  const fakeScriptTag1 = createdScriptTags[0];
-  // Call the error on the second script
-  fakeScriptTag1.onerror(new Error('Whoa there!'));
-
-  await expect(loadPromise).rejects.toThrowErrorMatchingInlineSnapshot(
-    `"Failed to load \\"plugin-a\\" bundle (/bundles/plugin/plugin-a/plugin-a.plugin.js)"`
-  );
-});
-
-test('`loadPluginBundles` rejects if timeout is reached', async () => {
-  await expect(
-    // Override the timeout to 1 ms for testi.
-    loadPluginBundle(addBasePath, 'plugin-a', { timeoutMs: 1 })
-  ).rejects.toThrowErrorMatchingInlineSnapshot(
-    `"Timeout reached when loading \\"plugin-a\\" bundle (/bundles/plugin/plugin-a/plugin-a.plugin.js)"`
-  );
-});
-
-test('`loadPluginBundles` rejects if bundle does attach an initializer to window.__kbnBundles__', async () => {
```

Comment:
I believe this test is still relevant to plugin_reader

Reply:
Agreed, I wrote a handful of tests for the plugin reader in https://github.com/elastic/kibana/pull/64685/commits/424ff74c5860ac64afe07f96055f3599b537b61b